### ISSUE TITLE
Completely Broken in getTraitChecklist function, but I'd love to help!

### ISSUE THREAD

**Issue by carsonbird** [zeel01/MonsterBlocks#228]
Recently, I think in the FoundryVTT 11 update, Monster Block stopped working. Whenever I attempt to open a monster sheet, nothing happens, and the console lists the following:

```
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at MonsterBlock5e.getTraitChecklist (MonsterBlock5e.js:394:10)
    at MonsterBlock5e.prepDamageTypeMenu (MonsterBlock5e.js:347:8)
    at MonsterBlock5e.prepAttributeMenu (MonsterBlock5e.js:297:21)
    at MonsterBlock5e.prepMenus (MonsterBlock5e.js:220:21)
    at new MonsterBlock5e (MonsterBlock5e.js:31:8)
    at get sheet (foundry.js:14122:23)
    at Token5e._onClickLeft2 (foundry.js:56267:29)
    at MouseInteractionManager.callback (foundry.js:31766:17)
    at #handleClickLeft2 (foundry.js:31971:15)
    at #handleMouseDown (foundry.js:31938:83)
    at Na.notifyListeners (earcut.js:128:75)
    at Na.notifyTarget (earcut.js:128:75)
    at Na.propagate (earcut.js:128:75)
    at Na.dispatchEvent (earcut.js:128:75)
    at Na.mapPointerDown (earcut.js:128:75)
    at Na.mapEvent (earcut.js:128:75)
    at yn.onPointerDown (earcut.js:128:75)
```

This points to [this line](https://github.com/zeel01/MonsterBlocks/blob/bf36d0e4f83a4dcd9059da39028e0c04d7304ce7/scripts/dnd5e/MonsterBlock5e.js#L394), stating that `traitList` is null or undefined, and not an object.

**Comment by carsonbird** [comment]
Oh, I think this is also related/the same as #219. Is this released? I'm using v3.4.0 and Foundry v11.315. Can I help get the next version released, if it's still pending? Are there a lot of cascading errors behind this one that are still being solved? @zeel01

**Comment by zeel01** [comment]
Yes, looks like this is the same as #219. An update has not yet been released, please see #224 for additional details.
